sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.execution.adaptive
 
 import java.io.File
+import java.lang.reflect.InvocationTargetException
```

Comment:
Please remove this since this is not the expected exception.

Reply:
Thanks for the review. The InvocationTargetException is here because the test is using reflection to invoke doExecute. This exception is wrapping the IllegalStateException. I will update the test tomorrow to make this clearer.

Reply:
Oh, got it. I missed that.